PNAS format

dhuppenkothen opened the pull request:
Exciting news: PNAS has a much nicer journal template than Nature, *and* they have an official LaTeX template.

I've so far only changed the style to the PNAS one, but not the layout of the paper, because I don't actually think we need to. I looked through the most recent PNAS issue, and found a bunch of papers that don't follow their official guidelines for which sections should go where, e.g. [this one](http://www.pnas.org/content/114/38/10244.full) with no sections at all, [this one](http://www.pnas.org/content/114/38/10035.full) and [this one](http://www.pnas.org/content/114/38/10029.full) with their own made-up section headings.

I've added a significance statement, too, so please have a look at that and tweak if you think it's necessary. Same goes for the "author contributions" statement, which I've taken a guess at, but I'm sure I have forgotten someone's contributions (sorry!). 

@arokem @aaarendt @davidwhogg @karthik @jakevdp: Please add/check your official affiliations (with address!) because for most of you I wasn't sure what this would be.

arokem commented on paper-pnas.tex:
```diff
@@ -0,0 +1,219 @@
+\affil[d]{Center for Computational Astrophysics, Flatiron Institute, 162 5th Ave, New York, NY 10010, USA}
+\affil[e]{}
+\affil[f]{}
+\affil[g]{}
```
The University of Washington eScience Institute, The WRF Data Science Studio, Physics/Astronomy Tower, 6th Floor, 3910 15th Ave NE,  Campus Box 351570
University of Washington, Seattle, WA 98105, USA

arokem commented on paper-pnas.tex:
```diff
@@ -0,0 +1,219 @@
+\leadauthor{Huppenkothen} 
+
+% Please add here a significance statement to explain the relevance of your work
+\significancestatement{As scientific disciplines grapple with more data sets of rapidly increasing complexity and size, new approaches are urgently required to introduce new statistical and computational tools into research communities and improve the cross-disciplinary exchange of ideas. In this paper, we introduce a new type of scientific workshop, called a \textit{hack week}, which allows for  ...
```
Super!

dhuppenkothen replied:
I feel like this should emphasize that this is useful for *all* sciences, but I couldn't quite put that into words. We have ~ 15 words left to the word limit, so if someone wants to put this into an even broader context, be my guest. :)

arokem commented on paper-pnas.tex:
```diff
@@ -0,0 +1,219 @@
+\significancestatement{As scientific disciplines grapple with more data sets of rapidly increasing complexity and size, new approaches are urgently required to introduce new statistical and computational tools into research communities and improve the cross-disciplinary exchange of ideas. In this paper, we introduce a new type of scientific workshop, called a \textit{hack week}, which allows for  ...
+
+% Please include corresponding author, author contribution and author declaration information
+\authorcontributions{The concept was designed by Jake VanderPlas, the paper design initially by David W. Hogg and Jake VanderPlas. Daniela Huppenkothen, Ariel Rokem and Anthony Arendt performed the studies and the data analysis. All authors contributed to the final manuscript.}
```
We still need an author contribution for Karthik

arokem commented:
I believe we can add back Figure 1

dhuppenkothen commented:
* Added Figure 1 back, but I remember Hogg had some comments about that figure. Who has the original file?
* Added a short methods section (please review!), because I think if we use the survey, we should give a little more information about it
* Checked character count: PNAS allows 6 pages, up to 39,000 characters, which we seem to be roughly within.

dhuppenkothen commented:
Based on commit messages, Karthik was part of the initial group that set up this repo and paper, so I've added him there.

jakevdp commented:
My affiliation is the same as Ariel's

arokem commented on paper-pnas.tex:
```diff
@@ -0,0 +1,220 @@
+\correspondingauthor{\textsuperscript{1}To whom correspondence should be addressed. E-mail: daniela.huppenkothen@nyu.edu}
+
+% Keywords are not mandatory, but authors are strongly encouraged to provide them. If provided, please include two to five keywords, separated by the pipe symbol, e.g:
+\keywords{Data science $|$ Education $|$ Reproducibility $|$ Interdisciplinary Collaboration $I$ Astronomy $|$ Neurosciences $|$ Geosciences} 
```
"Neurosciences" should be "Neuroscience"

Also: the delimiter between "Interdisciplinary Collaboration" and "Astronomy" should be a vertical line.

arokem commented:
In the Makefile, we need to change "paper" (on line 3) to "paper-pnas"

dhuppenkothen commented:
I forgot to note down what @aaarendt's suggestions were for a new title before I closed the chat window. 😳 
Feel free to suggest changes!

arokem commented on sections/design.tex:
```diff
@@ -11,7 +11,7 @@ \section*{Design considerations}
 Universities are an obvious first choice for hosting a hack week given their access to scientific resources, research support and infrastructure. 
 However traditional university lecture halls often do not meet the hack week criteria for interactive exchanges.
-Fortunately active learning teaching methods are creating new environments that foster interactivity through modified seating arrangements and integration of computer monitors at group workstations.
+Fortunately active learning teaching methods are creating new environments that foster interactivity through modified seating arrangements and integration of computer monitors at group workstations. There is a natural tension between keeping the group together and providing physically separated break-out spaces. On the one hand, conducting the workshop in a single large venue improves group cohes ...
```
I find this first sentence to be a bit cryptic. If I didn't know the specific case this refers to, this would just confuse me. I realize it was here before. How about: "Fortunately, many universities are experimenting with other types of spaces: for example, the adoption of active learning teaching methods [CITE: http://onlinelibrary.wiley.com/doi/10.1002/j.2168-9830.2004.tb00809.x/full] have led to the development of modular classrooms, where seating arrangements can be flexibly modified and group activities can be more readily undertaken".

dhuppenkothen replied:
👍

arokem commented:
Nailed it! 

Arial is ugly, but we don't need to go that far...

dhuppenkothen commented:
Going to merge this, since this is the version submitted to PNAS.
